src/component/helpers/array-utils.ts

```diff
@@ -33,3 +33,12 @@ export function ensureIsArray<T>(elements: (T | string)[] | T | string, acceptEm
   }
   return returnedArray.filter(value => value);
 }
+
+export function arraysAreIdentical<T>(array1: T[], array2: T[]): boolean {
+  return (
```

Comment:
you should add a null check as well as calling
arraysAreIdentical(null, null) results now in "Cannot read property 'length' of null"